src/main/scala/com/leagueprojecto/api/services/couchdb/DatabaseService.scala

```diff
@@ -42,54 +47,69 @@ class DatabaseService extends Actor with ActorLogging {
     case GetSummoner(region, name) =>
       val id = s"$region:$name"
 
-      summonerDb.docs.get[Summoner](id).attemptRun match {
+      val summoners = tryWithCircuitBreaker(summonerDb.docs.get[Summoner](id).attemptRun)
+      summoners match {
         case \/-(summonerDoc) =>
           log.info(s"Yay got summoner from Db: $summonerDoc")
           sender() ! SummonerResult(summonerDoc.doc)
         case -\/(CouchException(e: Error)) if e.status == NotFound =>
           log.info(s"No summoner found ($id) from Db")
-          sender() ! NoSummonerFound
+          sender() ! NoResult
         case -\/(e) =>
           log.error(s"Error retrieving summoner ($id) from Db")
-          println(e)
+          sender() ! NoResult
       }
 
     case GetMatches(region, summonerId, matchIds) =>
       val id = s"$region:$summonerId"
       val decoratedIds: Seq[String] = matchIds.map(key => s"$id:$key")
-      matchesDb.docs.getMany.queryIncludeDocsAllowMissing[MatchDetail](decoratedIds).attemptRun match {
+
+      val matches = tryWithCircuitBreaker(matchesDb.docs.getMany.queryIncludeDocsAllowMissing[MatchDetail](decoratedIds).attemptRun)
+
+      matches match {
         case \/-(matchesDocs) =>
           log.info(s"Yay, got ${matchesDocs.getDocsData.size} matches from Db")
           sender() ! MatchesResult(matchesDocs.getDocsData)
+        case -\/(CouchException(e: Error)) if e.status == NotFound =>
+          log.info(s"No matches found ($matchIds) from Db")
+          sender() ! NoResult
         case -\/(e) =>
           log.error(s"Error retrieving matches ($matchIds) from Db")
-          e.printStackTrace()
+          log.error(s"${e.getMessage}")
```

Comment:
Hmm, can't we just log the whole exception? 
Aside from that we should try to keep the exception handling the same way in all those methods.